jobs/influxdb/templates/bin/influxdb_ctl.erb

```diff
@@ -19,12 +19,18 @@ exec 2> >( tee -a /var/vcap/sys/log/influxdb_ctl.err.log | logger -p user.error
 ulimit -c unlimited
 ulimit -n <%= p("influxdb.max_open_fd") %>
 
-for DIR in $DATA_DIR $STORE_DIR $LOG_DIR $RUN_DIR; do
+for DIR in $DATA_DIR $LOG_DIR $RUN_DIR; do
```

Comment:
Why we do not need to `chown -R` `/var/vcap/store/influxdb`  ?

Reply:
because if you do it recursively it overwrites the owner/permissions of /var/vcap/store/influxdb/admin_password_file that as the PR description says, should be root:root/0600 (instead of vcap:vcap/755)